api/room/models.py

```diff
@@ -12,6 +12,7 @@
 from api.tag.models import Tag  # noqa: F401
 from utilities.validator import verify_calendar_id
 from api.devices.models import Devices # noqa F4
+from api.office_structure.models import OfficeStructure  # noqa f401
 
```

Comment:
You can remove this noqa here.
Also, this import is not being used in this file, that is why you may be getting unused import.

Reply:
Thanks, @kampkelly, This was failing before without the import but it has been removed.